src/site/site.xml

```diff
@@ -36,4 +36,10 @@ under the License.
       <item name="Configuring Source Plugin" href="/examples/configureplugin.html"/>
     </menu>
   </body>
+
+  <skin>
+    <groupId>org.apache.maven.skins</groupId>
+    <artifactId>maven-fluido-skin</artifactId>
+    <version>2.1.0</version>
+  </skin>
```

Comment:
should be taken from parent ....

Reply:
version removed so taken from parent

Reply:
We can remove whole tag for skin 😄

Reply:
> We can remove whole tag for skin 😄

When I created the PR yesterday the built failed due the missing skin, that's why I added it:
https://github.com/apache/maven-source-plugin/actions/runs/14449578502/job/40518647343

Will check later again, when back home.

Reply:
Local build with site works, let's see what CI says

Reply:
I tried using the newest version of maven (as you said beta-4+ is needed) and of the api-dependencies (as there is no 4.0.0-rc3 for each of them).

```
    <mavenVersion>4.0.0-rc-3</mavenVersion>
    <version.maven.api-meta>4.0.0-rc-1</version.maven.api-meta>
    <version.maven.api-impl>4.0.0-beta-5</version.maven.api-impl>
    <version.maven.api-model>4.0.0-rc-3</version.maven.api-model>
    <version.maven.api-di>4.0.0-rc-3</version.maven.api-di>
    <version.maven.api-core>4.0.0-rc-3</version.maven.api-core>
``` 

But this fails with a lot of compilation errors thrown by the API plugins. Can you please give me a hint what to do if adding the skin into the site.xml is not wanted?